src/mlte/measurement/cpu/local_process_cpu_utilization.py

```diff
@@ -202,6 +205,11 @@ def __call__(self, pid: int, poll_interval: int = 1) -> CPUStatistics:
             max=max(stats),
         )
 
+    @classmethod
+    def value(self) -> Type[CPUStatistics]:
```

Comment:
I am wondering if there is a way to automate this through some additional runtime introspection? Such that each measurement we implement doesn't need to provide a similar method that provides the class type object for the value that it produces. I might experiment with this a bit.

Reply:
It doesn't seem to be very straightfoward, but I guess it could be possible through instrospection of the __call__ method.

Reply:
Not a blocking comment.